src/cleanlab_tlm/utils/tlm_calibrated.py

```diff
@@ -21,9 +21,95 @@
 
 if TYPE_CHECKING:
     from collections.abc import Sequence
-
     from cleanlab_tlm.internal.types import TLMQualityPreset
 
+def _get_skops():
+    """Lazy import for skops to avoid unnecessary dependency."""
+    try:
+        import skops.io
+        return skops.io
+    except ImportError:
+        raise ImportError(
+            "The skops package is required for model serialization. "
+            "Please install it with: pip install skops"
+        )
+
+def save_tlm_calibrated_state(model: 'TLMCalibrated', custom_eval_options: Dict[str, Any], filename: str) -> None:
+    """Save fitted TLMCalibrated model state to file.
+    
+    Args:
+        model: A fitted TLMCalibrated model instance
+        custom_eval_options: Dictionary containing custom evaluation options
```

Comment:
Can we not directly access/save the options dict from `model._options` instead of having users specify this? I can foresee a lot of weird behaviour potentially happening if users specify invalid/different options here